src/org/opendatakit/briefcase/util/DatabaseUtils.java

```diff
@@ -95,6 +90,17 @@ private void assertRecordedInstanceTable() throws SQLException {
     }
   }
 
+  private void dumpRecordedInstanceTable() throws SQLException {
+    try (Statement stmt = connection.createStatement()) {
+      stmt.execute(SELECT_ALL_SQL);
+      try (ResultSet rs = stmt.getResultSet()) {
+        while (rs.next()) {
+          log.debug("recorded: " + rs.getString(1) + " @dir=" + rs.getString(2));
```

Comment:
@batkinson you mentioned quickly that you had noticed the return of a slowdown and I think it was based on this logging, is that right? The issue being that `isDebugEnabled` always true?

With my setup, I can't seem to get these debug statements to show up at all although `isDebugEnabled` does evaluate to true. If I put a `log.error` I can see it, though. In other classes, info messages show up at my IntelliJ console. I'm having a tough time following how logging is intended to work. @batkinson, if you're around for a quick chat in #briefcase-code, it might be helpful to talk live for a minute and strategize on how to approach getting more sane logging both for performance and general improvements.